bookwyrm/views/rss_feed.py

```diff
@@ -0,0 +1,29 @@
+''' '''
+
+from django.contrib.syndication.views import Feed
+from django.urls import reverse
+from bookwyrm.models.user import User
+from .helpers import get_activity_feed, get_user_from_username
+
+class RssFeed(Feed):
+
+    description_template = "snippets/rss_content.html"
+    title_template = "snippets/rss_title.html"
+
+    def get_object(self, request, username):
+        return get_user_from_username(username)
+
+    def link(self, obj):
+        return obj.local_path
+
+    def title(self, obj):
+        return f"Status updates from {obj.username}"
+
+
+    def items(self, obj):
+        return get_activity_feed(obj, ['public', 'unlisted', 'followers'])
```

Comment:
I wonder if this needs `followers`? I think (and please correct me if I'm wrong, I don't know much about rss) that this will always be called without an authenticated user?

Reply:
Yeah, I don't think we'll ever need authentication around the RSS feed. Honestly, I was just copying that from other parts of the code to get it to work.

Reply:
that's how I code too! if you remove followers from the list it should be good to go.